app/services/search_query_parser.rb

```diff
@@ -0,0 +1,22 @@
+class SearchQueryParser
+  API_URL = ENV['TARIFF_QUERY_SEARCH_PARSER_URL']
+
+  def initialize
+    @client = Faraday.new(
+      url: API_URL,
+      headers: {'Content-Type' => 'application/json'}
+      )
+  end
+
+  def get_tokens(search_query_term)
+    api_response = @client.get("tokens/#{search_query_term}")
```

Comment:
Should this be URL escaping the `search_query_term` - or does Faraday handle that?